src/scripts/main.js

```diff
@@ -1 +1,91 @@
 'use strict';
+
+const notification = document.createElement('div');
+
+notification.setAttribute('data-qa', 'notification');
+
+const promise1 = new Promise((resolve, reject) => {
+  let clickHappened = false;
+
+  const clickHandle = () => {
+    const textResolve = 'First promise was resolved';
+
+    notification.append(textResolve);
+
+    resolve(notification);
+    clickHappened = true;
+
+    document.removeEventListener('click', clickHandle);
+  };
+
+  document.addEventListener('click', clickHandle);
+
+  setTimeout(() => {
+    if (!clickHappened) {
+      const textReject = 'First promise was rejected';
+
+      notification.append(textReject);
```

Comment:
Similar to the previous comment, appending text directly to the notification element may result in concatenated messages. Consider updating the content instead.